build.gradle

```diff
@@ -18,32 +18,23 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  *
  */
-
-/**
-* The application plugin facilitates creating an executable JVM application.
-*/
+ 
+ /**
```

Comment:
This slash in the first line of a multi-line comment does not get indented with a space.

Reply:
Finished & pushed